src/main/java/employee/repository/EmployeeMongoRepository.java

```diff
@@ -0,0 +1,12 @@
+package employee.repository;
+
+import employee.data.EmployeeMongo;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+import java.util.Optional;
+
+@RepositoryRestResource(collectionResourceRel = "employees")
+public interface EmployeeMongoRepository extends MongoRepository<EmployeeMongo, Long> {
```

Comment:
If we named this `EmployeeMongoRepository` then we should name the other `EmployeeJpaRepository` or something similar.